src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/NameScope.cs

```diff
@@ -81,16 +73,8 @@ public void UnregisterName(string name)
             {
                 _nameMap.Remove(name);
             }
-            else
-            {
-                throw new ArgumentException(SR.Format(SR.NameScopeNameNotFound, name));
-            }
 
-            //if( TraceNameScope.IsEnabled )
-            //{
-            //    TraceNameScope.TraceActivityItem( TraceNameScope.UnregisterName,
-            //                                      this, name );
-            //}
+            throw new ArgumentException(SR.Format(SR.NameScopeNameNotFound, name));
```

Comment:
@hughbe, this method will always throw now !

Reply:
resolved